functions/Install-DbaMaintenanceSolution.ps1

```diff
@@ -216,8 +216,8 @@ function Install-DbaMaintenanceSolution {
 
             # Backup location
             if ($BackupLocation) {
-                $findBKP = 'C:\Backup'
-                $replaceBKP = $BackupLocation
+                $findBKP = 'SET @BackupDirectory     = NULL'
+                $replaceBKP = 'SET @BackupDirectory     = N''' + $BackupLocation + ''''
```

Comment:
Mmm what if BackupLocation is empty? I'd do a 

`if ((Test-Bound -ParameterName BackupLocation))`

Reply:
Already part of the code, earlier up. ;)

if ((Test-Bound -Parameter BackupLocation -Not)) {
                $BackupLocation = (Get-DbaDefaultPath -SqlInstance $server).Backup
            }

Reply:
aw, man. my bad. i looked at the code at the top but didnt look within the body!